ci/build.bat

```diff
@@ -1,27 +1,33 @@
-:: These are native MinGW builds, but they use the toolchain inside
-:: MSYS2, this allows using all the dependencies and tools available
-:: in MSYS2, but we cannot build inside the MSYS2 shell.
 echo on
 if "%CONFIGURATION%" == "MINGW_32" (
   set ARCH=i686
   set BITS=32
-) else (
+) else if "%CONFIGURATION%" == "MINGW_64" (
```

Comment:
This should stay as `else`. We need `ARCH` and `BITS` to be set for `MINGW_64-gcov`.